hw14.py

```diff
@@ -0,0 +1,33 @@
+from re import search
+phone_regex = '(?:\+\d{1,2})?\d{10}'

```

Comment:
Your pattern validates a lot of bad formats I tried. Here are some cases:
```
a0951234567 -> 0951234567
8+30951234567 -> +30951234567
a+380951234567 -> +380951234567
+380951234567a -> +380951234567
+3809512345677a -> +380951234567
+3809512345677 -> +380951234567
80951234567 -> 8095123456
+830965254141 -> +830965254141

Reply:
@BobJ0hnson the pattern still doesn't work properly. Try to change `search` to `full_match`, it will help to decrease the number of invalid inputs.
After that, you will need to fix only these invalid formats:
```
095123456
38095123456
```

You can simplify your pattern to something like this to make it work:
```
(\+?38)?0\d{9}
```